docs/evaluation/eval-repo.md

```diff
@@ -0,0 +1,49 @@
+# New Repository Proposal: eval
+
+## Summary
+
+This document proposes a new repository under the `instructlab` GitHub organization:
+
+- `instructlab/eval`
+
+## Background
+
+The `instructlab/instructlab` repository currently includes no real implementation
+of Evaluation as described by the [LAB paper](https://arxiv.org/abs/2403.01081). The
+closest implementation currently in `instructlab/instructlab` via the `ilab test` command.
```

Comment:
The final sentence can be cut. See next https://github.com/instructlab/dev-docs/pull/79/files#r1633921589

Reply:
See answers there